templates/style/component/_navigationBtnDesktop.html.twig

```diff
@@ -8,7 +8,7 @@
 			</a>
 		</div>
 		<div class="col mt-4">
-			<a class="btn" role="button" href="#">
+			<a class="btn" role="button" href="{{ path('candidate_applications', { id: candidate.id }) }}">
 				<h4>APPLICATIONS</h4>
 				<p>Find the status and the history of your applications.</p>
```

Comment:
'find your application status and history' plutôt

Reply:
ok!